packages/backend/db/schema.ts

```diff
@@ -15,3 +15,67 @@ export const noncesTable = pgTable('nonces', {
 });
 export type Nonce = typeof noncesTable.$inferSelect;
 export type NewNonce = typeof noncesTable.$inferInsert;
+
+export const privateTextsTable = pgTable('private_texts',{
+  id: uuid("id").default(crypto.randomUUID()).primaryKey().notNull(),
+  creator_id: uuid("creator_id").references(() => usersTable.id),
```

Comment:
creator_id is sometime useless. Because in blockchain, some users send transaction without our platform